style.css

```diff
@@ -305,9 +329,15 @@ h3 {
   /* font-size: 24px; */
 }
 
-.contact-title.desktop {
+/* .content {
+  background: url("/circle_header.svg") no-repeat;
+  background-position: 150% 150%;
+  background-size: 200px;
+} */
+
+/* .contact-title.desktop {
   display: block;
-}
+} */
```

Comment:
- [ ] Kindly remove the commented code, you can find it on the source tree always.